### ISSUE TITLE
ENH DICOM: Export capability

### ISSUE THREAD

**Issue by Lestropie** [MRtrix3/mrtrix3#2238]
*Very much* a wishlist item, but adding to the list nevertheless in case anyone encounters it and feels motivated to give it a try.

Reason I raise it now is I've been using `nifti2dicom` to convert BIDS Derivatives data to DICOM for PACS export, but due to omitting specific fields (most likely Window Centre / Width and/or SmallestImagePixelValue / LargestImagePixelValue), the image displayed on the hospital system changes brightness between slices. Could potentially be addressed through an update to `nifti2dicom`, but that software has not had an update in quite some time.

I would contemplate dedicating a command to DICOM export rather than having it as a read-write format within the default handling, as the latter would potentially lead to accidental DICOM exports due to omitting any file extension. Or at least restrict its usage to `mrconvert`.

**Comment by jdtournier** [comment]
Yes, it's something I've been meaning to look into. Might find the time in the New Year. Unlikely at this time...

**Comment by nayanw775** [comment]
@jdtournier @Lestropie That would be great if you can add this functionality, from a user point, since I have been using MRTRIX from past three and have extensively using to generate tracts for the pre-surgical planning, surgeon here always demands for the tract results in pacs, since we cannot afford a Navigation system of very quality such as Brainlab or Medtronics, they are pretty high costing in India so for the better planning it would be great.  Now we only take few pictures into the laptop or phone, for the guidance into the surgery.

**Comment by Lestropie** [comment]
@nayanw775 For tract export I would suggest looking at:

https://github.com/DevelopmentalImagingMCRI/karawun
